### ISSUE TITLE
How to uninstall?

### ISSUE THREAD

**Issue by faraco** [ch1x0r/LinuxRespin#10]
I downloaded the all.deb package and installed normally. However, when I type 'respin', nothing shows up. I also try to *uninstall* with 'sudo apt-get purge respin', yet the package is never showed up. So how can't I uninstall it?

**Comment by faraco** [comment]
Fixed it.

**Comment by ch1x0r** [comment]
Apt-get remove ...?  Is it installed?

**Comment by faraco** [comment]
Currently no. remove and purge doesn't work when I tried.

**Comment by ch1x0r** [comment]
So, since it seems you use sudo, what happens when you type in a shell sudo respin?

**Comment by ch1x0r** [comment]
So, I tested this in Ubuntu 16 today in a vm. I installed this - did need xresprobe. Then, it worked. See image 0 - working when I type sudo respin. Then, I uninstall this - see image removed-respin
![removerespin-0](https://cloud.githubusercontent.com/assets/4945600/24079661/234ffe9a-0c4a-11e7-815a-4bb7d87af11a.png)
![removed-respin](https://cloud.githubusercontent.com/assets/4945600/24079662/235337d6-0c4a-11e7-9d21-650a32439e6c.png)
